docs/02-guides/override-a-component.mdx

```diff
@@ -2,14 +2,177 @@
+let’s see how it works!
+
+Your own theme will be located in its own folder and will use default components
+from parent theme(s) — at least from Front-Commerce chocolatine theme. You would
+then copy the files you want to override in your theme folder by maintaining an
+identical file structure. Your component will then be used instead of the one in
+your parent theme(s).
+
+This translates in those three steps:
+
+1. configure your custom theme and use it in your application
+2. copy the file (`jsx`, `scss` or `gql`) you want to override in the `theme`
+   folder of the `app` directory
+3. customize its content in your extension directory
+
+## Creating the `theme` folder
+
+### Using the skeleton as base
+
+First we need to create the `theme` directory in your `app` folder
+
+```
+my-project
+└── app
+    └── theme
+```
+
+That’s it! You have successfully created the folder that will be the root of
+your custom theme !
+
+### In your own extension
+
+You can also extend theme in your extension, please read
+[Register an extension](/docs/remixed/guides/register-an-extension) guide to
+know how to create your own extension.
+
+First in your extension definition file, add the `theme` entry :
+
+```javascript title="extension/my-extension/index.ts"
+import path from "path";
+import { defineExtension } from "@front-commerce/core";
+
+export default function themeChocolatine() {
+  return defineExtension({
+    name: "MyExtension",
+    // highlight-next-line
+    theme: path.join(__dirname, "theme"),
+    configuration: {
+      providers: [],
+    },
+  });
+}
+```
+
+And now theme file will be injected in your project !
+
+## Override a component
+
+Let's add the description of a `Product` to a `ProductItem` as an example of
+overriding the base theme.
+
+The original file is:
+[`node_modules/@front-commerce/theme-chocolatine/theme/modules/ProductView/ProductItem/ProductItem.js`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/blob/main/packages/theme-chocolatine/theme/modules/ProductView/ProductItem/ProductItem.jsx)
+
+:::info
+
+Please refer to the
+[Front-Commerce’s folder structure documentation page](/docs/2.x/concepts/front-commerce-folder-structure)
+to get a better understanding of how components are organized in the base theme.
+
+:::
+
+1. copy it to:
+   `extension/my-extension/theme/ProductView/ProductItem/ProductItem.jsx` or
+   `app/theme/ProductView/ProductItem/ProductItem.jsx` (depending on the
+   [override method you've decided to use](#creating-the-theme-folder))
+2. add the description after the `<ProductOverview />` component in your
+   `ProductItem` with `{props.description}`.
+
+**But you are not done yet!** The `description` information is not included in
+the GraphQL fields fetched by the application in the base theme. You will thus
+need to update the fragment related to `ProductItem`.
+
+The original fragment file is collocated with the original component at:
```

Comment:
**thought:** As a first reader, it might be confusing to understand why it has to be this fragment that have to be overriden, even more now that we have two way of getting data (`routes` vs `useFetchers`) that don't use the same queries.

**suggestion:** Explain _quickly_ were does the data comes from (referencing other parts of the docs if needed) so that the updated fragment makes sense for the reader.

Reply:
Added a note